README.md

```diff
@@ -539,6 +539,33 @@ crud.len('customers')
 ...
 ```
 
+### Count
+
+```lua
+local result, err = crud.count(space_name, conditions, opts)
+```
+
```

Comment:
Done:

```diff
### Count

`CRUD` supports multi-conditional count, treating a cluster as a single space.
The same as with `select()` the conditions may include field names or numbers,
as well as index names. The recommended first condition is a TREE index; this
helps reducing the number of tuples to scan. Otherwise a full scan is performed.
+ If compared with `len()`, `count()` method works slower because `count()`
+ scans the entire space to count the tuples.
This method does yield that's why result may be approximate.
```

Reply:
As far as I know, we have a requirement to make a fast and rough estimate of tuples using `count()`. If count() is slower than len() why do we need count() at all?

Reply:
Updated:

```diff
### Count

`CRUD` supports multi-conditional count, treating a cluster as a single space.
The same as with `select()` the conditions may include field names or numbers,
as well as index names. The recommended first condition is a TREE index; this
helps reducing the number of tuples to scan. Otherwise a full scan is performed.
If compared with `len()`, `count()` method scans the entire space to count the
tuples according user conditions.
This method does yield that's why result may be approximate.
```